### PR TITLE
Make Dashboard page (front end)

### PR DESCRIPTION
**Pull request by Tomoki108**
***Commands to run***
-none

***Preconditions***
-access to the url (hostname/dashboard)

***Expected outputs***
-dashboard page is displayed as same as wireframe. 
https://docs.google.com/presentation/d/1a0mVYW0HPXVJn5pCqQp9an6AA6Kuq515HaVuDW9b030/edit#slide=id.g3cc2945e97_0_11

***asana***
-https://app.asana.com/0/984197702640917/984308420011935

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -18,3 +18,8 @@
 Auth::routes();
 
 Route::get('/home', 'HomeController@index')->name('home');
+
+//temporary
+Route::get('/dashboard', function () {
+    return view('dashboard');
```
Comment (jeremiahC) [inline routes/web.php:24]: will you create a controller for the dashboard in the future?

Comment (Tomoki108) [reply routes/web.php:24]: Okay, I'll make it

Comment (Tomoki108) [reply routes/web.php:24]: Yes I will. Then I don't have to do that in this PR?

File: resources/views/dashboard.blade.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
+@extends ('layouts.app')
+
+@section('content')
+<div class="container">
+  <div class="row">
```
Comment (jeremiahC) [inline resources/views/dashboard.blade.php]: please fix indentation from two space to four space, HTML should be four space.

Comment (jeremiahC) [reply resources/views/dashboard.blade.php]: there is a setting in vscode how to adjust space.

**Review by jeremiahC — APPROVED** [review]

**Pull request merged**